Fitness/Controllers/HomeController.swift

```diff
@@ -221,36 +264,40 @@ extension HomeController: UICollectionViewDataSource {
 extension HomeController: UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
 
     func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, referenceSizeForHeaderInSection section: Int) -> CGSize {
+        let mainCollectionViewHeights = 32.0
         if collectionView != mainCollectionView { return .zero }
         return CGSize(width: collectionView.bounds.width, height: 32.0)
     }
 
     func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
-        if collectionView != mainCollectionView {
-            return CGSize(width: 228.0, height: 195.0)}
+        if collectionView.accessibilityIdentifier == Identifiers.todaysClassesCell{
+            return getCollectionViewCellSize(collectionView, identifier: "classCell")
```

Comment:
same comment about moving these to the files that actually contain the collectionView